tests/e2e/pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2012-2018 Red Hat, Inc.
```

Comment:
Can you please help me how to update this copyright to 2021? If I replace 2018 with 2021, I get error:
`[WARNING] Missing header in: /home/.../che/tests/e2e/pom.xml`

Reply:
we still live in 2018 :)

Reply:
https://github.com/eclipse/che/issues/18821

Reply:
Ah, thanks for info :D